pages/Documents/MessagingChannels/ProactiveMessaging/api.md

```diff
@@ -270,4 +468,35 @@ Proactive Messaging service has retry mechanism internally on dependent services
 | skill             | 255 char max length | |
 | templateId | 32 char max length | templateId will be provided by proactive messaging when a template is created |
 | outboundNumber            | 16 char max length | Phone number cannot be more than 16 digit |
-| variable  		  | 1550 char max length | SMS messages more than 160 chars may be sent as multiple messages |
\ No newline at end of file
+| variable  		  | 1550 char max length | SMS messages more than 160 chars may be sent as multiple messages |
+
+<strong>Is Whatsapp Rich template enabled for all accounts?</strong>
+- To use Whatsapp Rich template via API/UI, reach out to Proactive team to get it enabled for the account. Currently the feature is not enabled for all accounts. It is enabled per account level.
+
+<strong>What are different components of a Whatsapp template? Are all the components mandatory to be present in a rich template?</strong>
+- There are four different components of a Whatsapp template
+1> Header (Image, Video, Document is supported currently. Only 1 type is allowed for a template)
+2> Body
+3> Footer
+4> Buttons (2 types: Quick Reply Buttons, Call To Action Buttons)
+A template will mandatorily have a message body; other components like header, footer, buttons may or may not be present for a template.
```

Comment:
can we change "Message body in WA Rich template is mandatory. Header, footer, buttons are optional in WA Rich template depending on template design."